.changeset/eleven-monkeys-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/checkbox': patch
```

Comment:
I think this would be a `minor` change, since it adds a feature

Reply:
Okay!! Should I change it manually to `minor` in this file or should I run the script again to change it?